code-challenges.js

```diff
@@ -57,12 +127,37 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
-// --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
+// console.log(myNumbers.length - 2)
+// console.log(myNumbers.length - 1)
+
+// console.log(myNumbers.lastIndexOf(givenValue1))
+// console.log(myNumbers.lastIndexOf(givenValue2))
+
+
+// The first code works, but I feel like I am not understanding how to use the variables "givenValue1, givenValue2" to run my code without subtracting from the total index of myNumbers. 
+// I went back and researched other options and found the .lastIndexOf method (from Javascripttutorials.net), and I was able to use all the given Data in my code.
```

Comment:
Great job on this one! I agree, I don't think the first approach is dynamic and wouldn't be a solution we could apply to many situations.